docs/index.rst

```diff
@@ -63,6 +63,15 @@ The following data attributes are supported on the ad placement element:
     This is useful if you want to conditionally load advertising for some users but not others
     or only load advertising when specific actions are performed.
 
+``data-ea-force-ad`` (optional)
+    This parameter can be used to test the ad client on a specific ad.
+    When used, any impressions will not be counted for billing purposes.
```

Comment:
This seems fine for now, but I could see some edge cases where we want this. The PSF for example wants to at least report on the views, though they won't be billed directly, so as long as we're enforcing that they're not paid ads on the server instead of not counting them, it should work for that use case.

Reply:
The server is already enforcing that "forced" ads are never counted. I don't think this will need to change for the PSF -- they'll have a custom ad type -- but it's something to keep in mind.